### ISSUE TITLE
Unexpected unilateral close: CHANNELD_NORMAL:bad reestablish dataloss msg

### ISSUE THREAD

**Issue by sgeisler** [ElementsProject/lightning#2461]
### Issue and Steps to Reproduce
I opened a channel to `036a54f02d2186de192e4bcec3f7b47adb43b1fa965793387cd2471990ce1d236b` yesterday and my node closed it unilaterally with the following channel status:

```
"CHANNELD_NORMAL:bad reestablish dataloss msg: WIRE_ANNOUNCEMENT_SIGNATURES 01031734e29c84af762e0ca98b5a1033e17ed0ac88b14bbbdb6ffd5662179ec5397b08a36700031e000156927af1ab354af056e0d354420806fb5040762ed0a964120f6d738f5c10d18549b969555451807d935ff8692bb33b1a336ff7ba206e1267f0354ba8dac69a4010347f03e28d6635568012166d42b634985d1e85052c8759f1fadaca635a8618606c3e333e0d406f3c21bcd6823308de316490d5a7ad2d5089368d2d0e8b99b1", 
"ONCHAIN:Tracking our own unilateral close", 
"ONCHAIN:2 outputs unresolved: in 119 blocks will spend DELAYED_OUTPUT_TO_US (8dfa3629cee1a371616d062c92876e4c03b7ed7739c6a30fb9ea7842b466557f:0) using OUR_DELAYED_RETURN_TO_WALLET"
```

### `getinfo` output
```
{
  "id": "021f5bfc335e949e555e93a1532b670a37ef7325410d2680fe5a8cf483eff161e2", 
  "alias": "bcwat.ch", 
  "color": "021f5b", 
  "num_peers": 23, 
  "num_pending_channels": 1, 
  "num_active_channels": 21, 
  "num_inactive_channels": 1, 
  "address": [
    {
      "type": "ipv4", 
      "address": "5.189.166.193", 
      "port": 9735
    }, 
    {
      "type": "torv2", 
      "address": "f3lt3sedgjnt6idt.onion", 
      "port": 9735
    }
  ], 
  "binding": [
    {
      "type": "ipv4", 
      "address": "5.189.166.193", 
      "port": 9735
    }
  ], 
  "version": "v0.7.0", 
  "blockheight": 566239, 
  "network": "bitcoin", 
  "msatoshi_fees_collected": ..., 
  "fees_collected_msat": "..."
}
```

**Comment by niftynei** [comment]
this is a known bug with lnd. see test in integration project: https://github.com/cdecker/lightning-integration/commit/4ece43cd37b864aeb4a72b5d7f84c7699bed49c6

**Comment by gallizoltan** [comment]
This is the worst bug I've ever seen. It closed 8 of my channels today. This makes the whole software unusable.

It took me months to build up lots of connection with enough inbound and outbound capacity. It is so frustrating to see as my node just closes them.

**Comment by mloop1** [comment]
I also had the same issue with 2 of my channels

**Comment by rustyrussell** [comment]
This was fixed in 0.7.1, FWIW: https://github.com/ElementsProject/lightning/issues/2559
